### PR TITLE
Roll emsdk install back to 3.1.7

### PR DESCRIPTION
**Pull request by lewing**

### CODE REVIEW

File: eng/Versions.props [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <PreReleaseVersionLabel>preview</PreReleaseVersionLabel>
     <PreReleaseVersionIteration>6</PreReleaseVersionIteration>
     <WorkloadSdkBandVersion>7.0.100</WorkloadSdkBandVersion>
-    <EmscriptenVersion>3.1.12</EmscriptenVersion>
+    <EmscriptenVersion>3.1.9</EmscriptenVersion>
```
Comment (radical) [inline eng/Versions.props:8]: 3.1.7?

Comment (radical) [inline eng/Versions.props:8]:
```suggestion
    <EmscriptenVersion>3.1.7</EmscriptenVersion>
```

**Review by radical — APPROVED** [review]

**Comment by radical** [comment]
@radekdoulik i can't merge this

**Review by radekdoulik — APPROVED** [review]

**Comment by akoeplinger** [comment]
@radical can you make sure you're part of this team, it should give you write access here: https://repos.opensource.microsoft.com/orgs/dotnet/teams/dotnet-corefx

**Comment by akoeplinger** [comment]
Ah looks like there was a branch protection set up so only admins could push, no idea why we had that, I disabled it.

**Pull request merged**